### ISSUE TITLE
Error building adguard-unbound-doh

### ISSUE THREAD

**Issue by JulesCyb** [notthebee/ansible-easy-vpn#242]
- [x] I've read the [FAQ](https://github.com/notthebee/ansible-easy-vpn/blob/main/FAQ.md) and my problem isn't described there.
- [x] I've also searched through the existing issues, and my problem hasn't been brought up

**Describe the issue**
The script fails on Ubuntu 22.04 with this error message.  It seems it can't reach internic.net, if I read the log correctly. But I do not know why it fails.

**Logs (if applicable)** 
Paste any relevant error messages here:
```
PLAY [Install and configure ansible-easy-vpn] *********************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] ********************************************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [Set the AWS and Letsencrypt staging facts] ******************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Update the packages and configure auto-updates] ****************************************************************************************************************************************************************************************************************
included: /root/ansible-easy-vpn/roles/system/tasks/setup.yml for localhost

TASK [system : Update and upgrade apt packages] *******************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Install unattended upgrades package] ***************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Copy unattended-upgrades configuration files in place] *********************************************************************************************************************************************************************************************************
changed: [localhost] => (item=10periodic)
changed: [localhost] => (item=50unattended-upgrades)

TASK [system : Configure the firewall] ****************************************************************************************************************************************************************************************************************************************
included: /root/ansible-easy-vpn/roles/system/tasks/firewall.yml for localhost

TASK [system : Template the iptables rules] ***********************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Copy the iptables systemd service] *****************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Enable and start the iptables service] *************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Configure the user account] ************************************************************************************************************************************************************************************************************************************
included: /root/ansible-easy-vpn/roles/system/tasks/user.yml for localhost

TASK [system : Ensure the root group exists] **********************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Allow sudo group to have passwordless sudo] ********************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Create a login user] *******************************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Ensure group "trust" exists] ***********************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Check if the Ansible playbook is copied to the new user directory] *********************************************************************************************************************************************************************************************
ok: [localhost]

TASK [system : Add the direnv hook to trust`s bashrc] *************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Copy the Ansible playbook to the new user] *********************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Fix the user's venv copy] **************************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [system : Chmod the ansible-easy-vpn directory] **************************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [system : Chmod the user home directory] *********************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : Load OS-specific vars.] ****************************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : include_tasks] *************************************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : include_tasks] *************************************************************************************************************************************************************************************************************************************
included: /root/ansible-easy-vpn/.ansible/roles/geerlingguy.docker/tasks/setup-Debian.yml for localhost

TASK [geerlingguy.docker : Ensure old versions of Docker are not installed.] **************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : Ensure dependencies are installed.] ****************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : Ensure additional dependencies are installed (on Ubuntu < 20.04 and any other systems).] ***********************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Ensure additional dependencies are installed (on Ubuntu >= 20.04).] ********************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : Add Docker apt key.] *******************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : Ensure curl is present (on older systems without SNI).] ********************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Add Docker apt key (alternative for older systems without SNI).] ***********************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Add Docker repository.] ****************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : Install Docker packages.] **************************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Install Docker packages (with downgrade option).] **************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : Install docker-compose plugin.] ********************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Install docker-compose-plugin (with downgrade option).] ********************************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : Ensure /etc/docker/ directory exists.] *************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Configure Docker daemon options.] ******************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Ensure Docker is started and enabled at boot.] *****************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : Ensure handlers are notified now to avoid firewall conflicts.] *************************************************************************************************************************************************************************************

RUNNING HANDLER [geerlingguy.docker : restart docker] *************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [geerlingguy.docker : include_tasks] *************************************************************************************************************************************************************************************************************************************
skipping: [localhost]

TASK [geerlingguy.docker : Get docker group info using getent.] ***************************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [geerlingguy.docker : Check if there are any users to add to the docker group.] ******************************************************************************************************************************************************************************************
ok: [localhost] => (item=trust)

TASK [geerlingguy.docker : include_tasks] *************************************************************************************************************************************************************************************************************************************
included: /root/ansible-easy-vpn/.ansible/roles/geerlingguy.docker/tasks/docker-users.yml for localhost

TASK [geerlingguy.docker : Ensure docker users are added to the docker group.] ************************************************************************************************************************************************************************************************
changed: [localhost] => (item=trust)

TASK [geerlingguy.docker : Reset ssh connection to apply user changes.] *******************************************************************************************************************************************************************************************************
[WARNING]: Reset is not implemented for this connection

TASK [docker_network : Create the wg network] *********************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [fail2ban : Install fail2ban] ********************************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [fail2ban : Disable e-mail notifications on jail stop and start] *********************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [fail2ban : Install the jail.local file] *********************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [fail2ban : Make sure the fail2ban systemd service is enabled and started] ***********************************************************************************************************************************************************************************************
changed: [localhost]

TASK [dns : Create the adguard-unbound-doh config folders] ********************************************************************************************************************************************************************************************************************
changed: [localhost] => (item=adguard-unbound-doh)
changed: [localhost] => (item=adguard-unbound-doh/adguard)
changed: [localhost] => (item=adguard-unbound-doh/build)
changed: [localhost] => (item=adguard-unbound-doh/dnscrypt-proxy)
changed: [localhost] => (item=adguard-unbound-doh/adguard/work)
changed: [localhost] => (item=adguard-unbound-doh/adguard/conf)

TASK [dns : Copy the docker folder to /opt/docker] ****************************************************************************************************************************************************************************************************************************
changed: [localhost]

TASK [dns : Build the adguard-unbound Docker image] ***************************************************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => changed=false
  msg: 'Error building adguard-unbound-doh - code: 1, message: The command ''/bin/sh -c wget https://www.internic.net/domain/named.root -qO- >> /etc/unbound/root.hints'' returned a non-zero code: 1, logs: [''Step 1/11 : FROM alpine:3.15'', '''', ''Pulling from library/alpine'', ''Pulling fs layer'', ''Downloading'', ''Downloading'', ''Verifying Checksum'', ''Download complete'', ''Extracting'', ''Extracting'', ''Extracting'', ''Pull complete'', ''Digest: sha256:794bc829ea39d85926be70fedfc9fb17aa4244d82b2fbecb5634b03e5dd97aca'', ''Status: Downloaded newer image for alpine:3.15'', '' ---> d51a1e6a80db'', ''Step 2/11 : RUN apk add --no-cache         libcap         unbound=1.13.2-r2         dnscrypt-proxy'', '''', '' ---> Running in 641e402052cb'', ''fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz'', ''fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz'', ''(1/8) Installing ca-certificates (20230506-r0)'', ''(2/8) Installing dnscrypt-proxy
    (2.1.1-r2)'', ''Executing dnscrypt-proxy-2.1.1-r2.pre-install'', ''(3/8) Installing libcap (2.61-r1)'', ''(4/8) Installing dnssec-root (20190225-r1)'', ''(5/8) Installing libevent (2.1.12-r4)'', ''(6/8) Installing expat (2.5.0-r0)'', ''(7/8) Installing unbound-libs (1.13.2-r2)'', ''(8/8) Installing unbound (1.13.2-r2)'', ''Executing unbound-1.13.2-r2.pre-install'', ''Executing busybox-1.34.1-r7.trigger'', ''Executing ca-certificates-20230506-r0.trigger'', ''OK: 23 MiB in 22 packages'', ''Removing intermediate container 641e402052cb'', '' ---> f6072b5ce19c'', ''Step 3/11 : WORKDIR /tmp'', '''', '' ---> Running in 61b127b2e42c'', ''Removing intermediate container 61b127b2e42c'', '' ---> b52b566d3051'', ''Step 4/11 : RUN wget https://www.internic.net/domain/named.root -qO- >> /etc/unbound/root.hints'', '''', '' ---> Running in 398fdae27196'', "\x1b[91mwget: bad address ''www.internic.net''", ''\x1b[0m'', ''Removing intermediate container 398fdae27196'']'

PLAY RECAP ********************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=42   changed=22   unreachable=0    failed=1    skipped=10   rescued=0    ignored=0
```

**Environment:**
- Terminal
- Ubuntu 22.04
- Homelab Proxmox with static IP

**Comment by JulesCyb** [comment]
I ran the playbook in dozens of different machines, with different ports open and I got pretty frustrated. As I could install docker, portainer and then the containers myself manually, but not via script. 

I gave it a last try and installed it the one way I did not try before: With E-Mail functionality enabled. Everything ran perfect, no errors and everything works like a charm. 

I tried it on 2 more VMs and could repeat the successful and unsucessful installation. With and without E-Mail functionality.

Maybe this helps to pinpoint the problem.

This issue can be closed. THank you for your work and help. 

Schöne Grüße

**Comment by JulesCyb** [comment]
I ran the playbook in dozens of different machines, with different ports open and I got pretty frustrated. As I could install docker, portainer and then the containers myself manually, but not via script. 

I gave it a last try and installed it the one way I did not try before: With E-Mail functionality enabled. Everything ran perfect, no errors and everything works like a charm. 

I tried it on 2 more VMs and could repeat the successful and unsucessful installation. With and without E-Mail functionality.

Maybe this helps to pinpoint the problem.

This issue can be closed. THank you for your work and help. 

Schöne Grüße

**Comment by JulesCyb** [comment]
I ran the playbook in dozens of different machines, with different ports open and I got pretty frustrated. As I could install docker, portainer and then the containers myself manually, but not via script. 

I gave it a last try and installed it the one way I did not try before: With E-Mail functionality enabled. Everything ran perfect, no errors and everything works like a charm. 

I tried it on 2 more VMs and could repeat the successful and unsucessful installation. With and without E-Mail functionality.

Maybe this helps to pinpoint the problem.

This issue can be closed. THank you for your work and help. 

Schöne Grüße
